### PR TITLE
feat: add switch and slider radius

### PR DESCRIPTION
**Pull request by rubiin**
Lets user control the border radius for slider and switch , helping more on customization

For switch:
image

For slider:
image

### CODE REVIEW

File: widget/settings/pages/theme/menus/index.ts [hunk 1]
```diff
@@ -123,6 +129,16 @@ export const MenuTheme = (): Scrollable<Child, Attribute> => {
                     title: 'Background (Hover)',
                     type: 'color',
                 }),
+                Option({
+                    opt: options.theme.bar.menus.slider.slider_radius,
+                    title: 'Slider Border Radius',
```
Comment (Jas-SinghFSU) [inline widget/settings/pages/theme/menus/index.ts:134]:
```suggestion
                    title: 'Slider Puck Radius',
```

File: widget/settings/pages/theme/menus/index.ts [hunk 2]
```diff
@@ -123,6 +129,16 @@ export const MenuTheme = (): Scrollable<Child, Attribute> => {
...
+                    type: 'string',
+                }),
+                Option({
+                    opt: options.theme.bar.menus.slider.progress_radius,
+                    title: 'Slider Progress Radius',
```
Comment (Jas-SinghFSU) [inline widget/settings/pages/theme/menus/index.ts:139]:
```suggestion
                    title: 'Slider/Progress Bar Radius',
```

File: widget/settings/pages/theme/menus/index.ts [hunk 3]
```diff
@@ -91,6 +91,12 @@ export const MenuTheme = (): Scrollable<Child, Attribute> => {
                 Header('Switch'),
                 Option({ opt: options.theme.bar.menus.switch.enabled, title: 'Enabled', type: 'color' }),
                 Option({ opt: options.theme.bar.menus.switch.disabled, title: 'Disabled', type: 'color' }),
+                Option({ opt: options.theme.bar.menus.switch.radius, title: 'Border Radius', type: 'string' }),
+                Option({
+                    opt: options.theme.bar.menus.switch.slider_radius,
+                    title: 'Slider Border Radius',
```
Comment (Jas-SinghFSU) [inline widget/settings/pages/theme/menus/index.ts:97]:
```suggestion
                    title: 'Switch Puck Radius',
```

File: widget/settings/pages/theme/menus/index.ts [hunk 4]
```diff
@@ -91,6 +91,12 @@ export const MenuTheme = (): Scrollable<Child, Attribute> => {
                 Header('Switch'),
                 Option({ opt: options.theme.bar.menus.switch.enabled, title: 'Enabled', type: 'color' }),
                 Option({ opt: options.theme.bar.menus.switch.disabled, title: 'Disabled', type: 'color' }),
+                Option({ opt: options.theme.bar.menus.switch.radius, title: 'Border Radius', type: 'string' }),
```
Comment (Jas-SinghFSU) [inline widget/settings/pages/theme/menus/index.ts:94]:
```suggestion
                Option({ opt: options.theme.bar.menus.switch.radius, title: 'Switch Radius', type: 'string' }),
```

**Comment by Jas-SinghFSU** [comment]
Good stuff! There are other elements that should inherit this as well, things like the bar in the media player, the stats monitors in the dashboard, the brightness slider in the battery menu etc.

I have created a branch to add those as well. Could you merge branch `slider_radius` with this PR branch please? We should be good to merge after that and renaming some of the options as suggested above.

**Comment by rubiin** [comment]
done

**Comment by rubiin** [comment]
no conflict in local but here it shows conflict when pushing the merged changes

**Comment by Jas-SinghFSU** [comment]
You should be safe to consume incoming changes when resolving the conflict. That'll bring you up to date.

**Comment by rubiin** [comment]
git pull / push shows no changes

**Comment by Jas-SinghFSU** [comment]
During the merge of origin 'master' into `css`, when you pick which code to choose while resolving conflicts, are you choosing 'Current' over 'Incoming'? You should select `incoming` otherwise that seems to be making the code in origin/master get deleted. Notice the changes at the `dashboard.scss` getting removed which were added yesterday to auto-set margins for the dashboard.

**Comment by rubiin** [comment]
What i am doing is resetting to the hash, merging the branch slider_radius. But the conflict pops up.

**Comment by rubiin** [comment]
This time i reset to the hash, merged upstream/master and then the slider_radius. Got empty conflicts on local as well. 
Could you check the changes

**Comment by Jas-SinghFSU** [comment]
I've pre-resolved the conflicts in my branch. You should be ok to revert back the the safe commit hash and re-merge my branch into yours without conflicts.

**Comment by Jas-SinghFSU** [comment]
Niice! There it is, thanks for sticking through.

**Comment by rubiin** [comment]
no worries. had time to kill

**Pull request merged**